rxmagneto/src/main/java/com/aritraroy/rxmagneto/utils/Constants.java

```diff
@@ -1,10 +0,0 @@
-package com.aritraroy.rxmagneto.utils;
```

Comment:
Why are these files deleted? You shouldn't remove any already available functionality from the library.